src/manager/node.h

```diff
@@ -50,6 +50,7 @@ struct Node {
         LIST_HEAD(ProxyMonitor, proxy_monitors);
         LIST_HEAD(ProxyDependency, proxy_dependencies);
 
+        LIST_HEAD(Subscription, wildcard_subscriptions);
```

Comment:
Alternatively you could store the wildcard subscription in the unit_subscription hashmap under the "*" key. Not sure which is cleaner.

Reply:
I tried that approach and I could simplify a bit of code - so I'd go with that.